requirements.txt

```diff
@@ -11,6 +11,7 @@ python-dateutil==2.5.3
 pytz==2016.4
 requests==2.10.0
 six==1.10.0
+jsonschema==2.5.1
```

Comment:
I think you would need to add this (modified in some way to make it less specific) to the setup.py also